### ISSUE TITLE
six.moves.cPickle reported as import error

### ISSUE THREAD

**Issue by nebulans** [PyCQA/pylint#2185]
### Steps to reproduce
1. Create virtualenv with pylint and six
2. Create `example.py` with the following content

```python
import six.moves.cPickle as pickle
```

3. Run `pylint example.py`

### Current behavior

Output is 

```
************* Module example
C:  1, 0: Missing module docstring (missing-docstring)
E:  1, 0: No name 'cPickle' in module 'Moves' (no-name-in-module)
E:  1, 0: Unable to import 'six.moves.cPickle' (import-error)
W:  1, 0: Unused six.moves.cPickle imported as pickle (unused-import)
```

### Expected behavior

Output should not contain `import-error` or `no-name-in-module` as this import does in fact resolve

### pylint --version output

```
pylint 1.9.2, 
astroid 1.6.5
Python 2.7.12 (default, Feb 13 2018, 14:32:37) 
[GCC 5.4.0 20160609]
```

**Comment by brycepg** [comment]
Thanks for the report!

**Comment by PCManticore** [comment]
This should be available once we release 1.6!

**Comment by nebulans** [comment]
:+1: 

@PCManticore version 1.6 of what?

**Comment by PCManticore** [comment]
Sorry, of astroid!
